packages/app/lib/models/sightings.dart

```diff
@@ -70,6 +70,25 @@ class Sighting {
       String? comment,
       List<Species>? species,
       List<LocalName>? localNames}) async {
+    Species? updatedSpecies = species?.firstOrNull;
+    LocalName? updatedLocalName = localNames?.firstOrNull;
+
+    this.viewId = await updateSighting(this.viewId,
+        datetime: datetime,
+        latitude: latitude,
+        longitude: longitude,
+        comment: comment,
+        speciesIds: species == null
+            ? null
+            : updatedSpecies == null
+                ? []
+                : [updatedSpecies.id],
+        localNameIds: localNames == null
+            ? null
+            : updatedLocalName == null
+                ? []
+                : [updatedLocalName.id]);
```

Comment:
not a fan of the nested ternary - will rewrite to be closer to the original implementation

Reply:
addressed via https://github.com/p2panda/meli/pull/60/commits/585e5a22e79d94ada48dd0f794901cce2f572745